pkg/cli/install.go

```diff
@@ -8,6 +8,10 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	DEFAULT_WORKLOAD_MEMORY = int64(536870912) // 512M
```

Comment:
Don't use ALL_CAPS syntax.  Go style prefers `UpperCamelCase`  Also if it's not public use normal `camelCase`.  In short constants don't have a special case style.